Scripts/updateDependency.sh

```diff
@@ -67,3 +67,8 @@ do
 done
 
 rm -rf $dependency_directory
+s
+if [[ $dependency_directory == *"DifferenceKit"* ]]; then
+    # We currently use customized UIKit extensions in Utils/DifferenceKit+Stream.swift
+    rm $output_directory/Extensions/UIKitExtension.swift
+fi
```

Comment:
I added this for making sure we keep using the customised extension which is a copy of this file. Can be cleaned up if we manage to get the change into `DifferenceKit` itself